app/stylesheet/miq-structured-list.scss

```diff
@@ -123,12 +121,19 @@
       }
 
       i {
+        width: 20px;
+        height: 20px;
         font-size: 15px;
         display: flex;
         justify-content: center;
         flex-direction: row;
         padding: 5px;
       }
+      .fa-5x {
+        width: 70px !important;
+        height: 80px !important;
+        font-size: 5em !important;
+      }
```

Comment:
could you please add a custom class name `.fa-5x` to target the css for settings tab pages?